api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimServiceTest.java

```diff
@@ -520,6 +522,23 @@ void shouldSaveClaimantWhenEligibilityThrowsException() {
         assertThat(eventCaptor.getValue().getEventMetadata().get(FailureEvent.FAILED_EVENT_KEY)).isEqualTo(ClaimEventType.NEW_CLAIM);
         verifyNoInteractions(claimMessageSender);
     }
+    
+    @Test
+    void shouldUpdateClaimWithCurrentIdentityAndEligibilityResponse() {
+        // given
+        Claim claim = aValidClaim();
+        CombinedIdentityAndEligibilityResponse response = 
```

Comment:
Any particular reason you didn't use a test data factory method here? It can be any `CombinedIdentityAndEligibilityResponse` so seems like using one of the methods from there would be fine.

Reply:
only because we couldn't find it in eclipse ('Identify' was shortened to 'Id' in the factory name). We've used the factory now